src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -312,6 +312,21 @@
         <target state="translated">Pokud vytváříte cestu pro {0} v metodě {1} z relativní cesty k položce archivu za účelem extrahování souboru a daný zdroj je nedůvěryhodný archiv zip, zajistěte sanitizaci relativní cesty k položce archivu {2} v metodě {3}.</target>
         <note />
       </trans-unit>
+      <trans-unit id="DoNotAddFinalizerToMemoryManagedTDerivedTypesDescription">
```

Comment:
If it wasn't obvious to you, all of the .xlf files get generated from the resx, so you don't need to ever edit them, just make sure they get included in your commit after rebuilding.

Reply:
(I say this because it wsan't obvious to me at first)

Reply:
yeah, I didn't updated them at all, I was thinking to add them after wording get better, but it was generated anyways